andino_navigation/params/nav2_params.yaml

```diff
@@ -0,0 +1,350 @@
+amcl:
```

Comment:
Maybe is easy to make all this package Apache? @francocipollone

Reply:
It's not a matter of changing this license (BSD 3-Clause), the requirement is to comply with that the dependent package or used asset's license requires you to do.

Once again, to the best of my knowledge, here and in any other file which can be considered as a copy with modifications, you should do the following:

- State in a comment the copyright.
- State in a comment the license note (the whole license note)
- State in a comment the modifications introduced after those.
- It would be useful also to have a URL with where it was copied from. Preferably, it should in include the commit SHA.


If I had to do it, I would do something like:

```yaml
# This file has been copied and adapted from: https://github.com/ros-planning/navigation2/blob/3bb9a95719539fcdfedf520c63339bc3157f80b4/nav2_bringup/params/nav2_params.yaml
# 
# Copyright could not be identified. Copyright holders identification should refer to the aforementioned code source.
# 
# License note follows:
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
# 
#    http://www.apache.org/licenses/LICENSE-2.0
# 
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
# 
# Changes introduced to the original file are:
# ....
```

And then include a `NOTICE` file at the top level of your package with the following text:

```
This software contains code derived from the navigation2 project.
https://github.com/ros-planning/navigation2
In particular, refer to:  https://github.com/ros-planning/navigation2/blob/3bb9a95719539fcdfedf520c63339bc3157f80b4/nav2_bringup/params/nav2_params.yaml
```